### ISSUE TITLE
Hi, I'm having an overfit problem

### ISSUE THREAD

**Issue by xuzhuochang** [HappyColor/SpeechFormer#7]
I ran your code on the DAIC data set, the model is SpeechFormer-S, the audio features are logmel, and there is overfitting. Specifically, the loss in the training phase can be reduced, but the loss in the testing phase is getting higher and higher. Could you please give me some suggestions?
